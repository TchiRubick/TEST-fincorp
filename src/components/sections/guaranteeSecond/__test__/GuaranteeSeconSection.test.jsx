import React from 'react';
import ReactDom from 'react-dom';
import { cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

import GuaranteeSeconSection from '../GuaranteeSeconSection';

afterEach(cleanup);

it('renders good', () => {
  const div = document.createElement('div');

  ReactDom.render(<GuaranteeSeconSection />, div);
});

it('as expected snapshot 1', () => {
  const tree = renderer.create(<GuaranteeSeconSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
