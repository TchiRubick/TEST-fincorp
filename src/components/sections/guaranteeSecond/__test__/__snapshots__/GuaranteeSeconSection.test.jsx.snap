// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`as expected snapshot 1 1`] = `
<div
  className="comp-container"
>
  <div
    className="section-guarantee-second"
  >
    <div
      className="section-guarantee-second__block"
    >
      <img
        alt="shield"
        src="shield-1.png"
      />
      <div
        className="section-guarantee-second__block-text"
      >
        <span
          className="section-guarantee-second__block-text__title"
        >
          Insurance made easy
        </span>
        <span
          className="section-guarantee-second__block-text__description"
        >
          At fincorp, we are commited to provide top-notch services to our customers.
        </span>
      </div>
      <div
        className="section-guarantee-second__block-text__button"
      >
        <button
          className="comp-button-base __green"
          type="button"
        >
          Get your free Quote
        </button>
      </div>
    </div>
  </div>
</div>
`;
